### DISCUSSION TITLE
Trying to use PhantomJS on CircleCI Ubuntu

### DISCUSSION THREAD

**Discussion by safetypins** [macbre/phantomas#978] [Q&A]
I'm trying to use PhantomJS on an Ubuntu CircleCI image, but I'm getting the following error:

`phantomjs: error while loading shared libraries: libjpeg.so.8: cannot open shared object file: No such file or directory`

I found an older issue related to this error, but there wasn't a solution posted there, just that on a clean install the problem went away. But when I run `ldd phantomjs` it says "not a dynamic executable". This error happens when I run `phantomjs -v`. Any ideas of how I can isolate what is causing this?

**Comment by safetypins** [comment]
sorry, it looks like i posted this on the wrong project.

**Reply by macbre** [reply]
Now worries.
